src/main/java/nugget/Storage.java

```diff
@@ -0,0 +1,81 @@
+package nugget;
```

Comment:
Not sure where to put this,  but I am thinking would it be better to separate the classes to separate sub packages? Maybe something like nugget.exception and nugget.command?